src/enkf_chgres_recenter_nc.fd/CMakeLists.txt

```diff
@@ -10,6 +10,9 @@ utils.f90
 if(CMAKE_Fortran_COMPILER_ID MATCHES "^(Intel)$")
   set(CMAKE_Fortran_FLAGS "${CMAKE_Fortran_FLAGS} -fp-model precise")
 endif()
+if(ip_VERSION VERSION_GREATER_EQUAL 4.0.0)
+  set(CMAKE_Fortran_FLAGS "${CMAKE_Fortran_FLAGS} -DIP_V4")
+endif()
```

Comment:
Was `target_compile_definitions` not working? 
`CMAKE_Fortran_FLAGS` are typically compiler flags, while compile definitions are preprocessor directives.